config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -3194,7 +3194,7 @@ presubmits:
     skip_branches:
     - release-\d+\.\d+  # per-release image
     annotations:
-      testgrid-dashboards: sig-node-presubmits
+      testgrid-dashboards: sig-node-presubmits, sig-node-dynamic-resource-allocation
```

Comment:
Would it be okay to move the e2e-node-*dra jobs into `dynamic-resource-allocation.yaml`? I frequently got lost when editing the large `sig-node-presubmit.yaml`.

Of course that then makes the `sig-node-presubmit.yaml` file "incomplete".

Reply:
Split it until multiple presubmit files?

that's really up to you and the SIG, we gzip on upload to prow and I personally find I have to search the repo anyhow because this isn't reliable (see also the massive branch-forked jobs files)

Reply:
Yes, this was mostly a question for @kannon92.

Reply:
> Split it until multiple presubmit files?

That would be my vote. I like the separation of presubmits from periodics. If you want a dedicated file for DRA presubmits, that would be fine.